indico/modules/events/layout/default.py

```diff
@@ -143,15 +143,15 @@ def get_default_menu_entries():
         MenuEntryData(
```

Comment:
move to `indico/modules/events/contributions/__init__.py` (signal is already registered in the bottom for the contribution menu item)